### PR TITLE
Add description on building backend for development

### LINKED ISSUE #16: Update readme.md
The current readme.md is a template and doesn't describe the project.

### PR DESCRIPTION
**Pull request by BranAndSceolan**
Add badge showing codeql-analysis result

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+## Building This Project
+
+### Backend
+* go into the backend directory ``cd backend``
+* make sure all dependencies are installed ``npm install``
+* either:
+  * build for development: 
+    * create a .env file with
+      * NOTES_PASSWORD=choose_a_password
+      * POSTGRES_USER=choose_an_admin_name
```
Comment (danielp0505) [inline README.md:21]: Why does the backend need/use the admin account?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+      * POSTGRES_PASSWORD=choose_an_admin_password
+      * POSTGRES_DB=choose_a_db_name
+      * NODE_ENV=debug
+      * NOTES_USER=choose_a_username
+    * Replace all choose_-entries with the values you would like to use.
+    * source your .env file  ``source <path to .env>``
```
Comment (danielp0505) [inline README.md]: This is an incomplete command, is it a parameter?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+    * execute the docker-compose file in the root directory. ``docker-compose build``
```
Comment (danielp0505) [inline README.md]: This just builds images and doesn't start a database container if this is what you're referring to here. https://docs.docker.com/engine/reference/commandline/compose_build/
docker-compose up should automatically build the images and create containers as specified in the docker-compose.yml.

**Review by danielp0505 — CHANGES_REQUESTED** [review]
The instructions are incomplete/incorrect. Please see the comments.

File: README.md (hunk 2 above)
Comment (BranAndSceolan) [reply README.md]: yes, <path to .env> is intended as a placeholder for a parameter here.

File: README.md (hunk 1 above)
Comment (BranAndSceolan) [reply README.md:21]: The init-script used in the Dockerfile uses the admin account, so I thought backend needs it to construct the database container.

File: README.md (hunk 3 above)
Comment (BranAndSceolan) [reply README.md]: I replaced that part with cocker-compose up like recommended

**Review by danielp0505 — APPROVED** [review]

**Pull request merged**